tests/test_decay.py

```diff
@@ -61,11 +61,7 @@ def setup():
     350830000,  # decay to metastables without reason.
     340830000,  # ...
     481170000,  # ...
-    471110001,  # missing branch in origen
-    501130001,  # ...
-    561330001,  # FIXME These have too much mass on our end
-    491180000,  # ...
-    801970001,  # ...
```

Comment:
yep, after building `data.cpp` with this pr with a fresh nuc_data.h5 these tests should no need to be blacklisted.